students/Wayne/Assignments/Lesson3/Mailroom.py

```diff
@@ -1,51 +1,84 @@
 """
-Simple mailroom program for a non profit 
+Simple mailroom program for a non profit
 """
 
-#Start with interactive loop 
+# Start with interactive loop
 
 import sys
+from textwrap import dedent
 
-donors = [("Donor A",[50,100,24,100]),
-         ("Donor B",[525,10,23]),
-         ("Donor C",[50]),
-        ]
+donors = [("Donor A", [50, 100, 24, 100]),
```

Comment:
Cool, this looks a lot like a dict, so this should be an easy conversion when you get to part 2